src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -33,26 +33,26 @@
                   If the {{ isProprietor ? 'proprietor' : 'partner' }} has changed their legal name,
                   enter their new legal name.
                 </p>
-                <article class="form__row three-column pt-6">
+                <article class="form__row three-column pt-4">
                   <v-text-field
                     filled
-                    class="item"
+                    class="item mx-2 mb-n6"
```

Comment:
Note the negative bottom margin, which is to remove the extra whitespace (added by Vuetify for messages) below this component, so that subsequent elements can specific whatever top whitespace they want.

Rationale: On a page, you can decide that components will add whitespace after themselves, or you can decide that components will add whitespace before themselves. I prefer the latter as I find it easier to lay out components nicely when I only have to think about what is above the element I'm working on.